httpretty/core.py

```diff
@@ -102,6 +102,13 @@
 except ImportError:  # pragma: no cover
     ssl = None
 
+# used to handle error caused by ndg-httpsclient
+try:
+    from requests.packages.urllib3.contrib.pyopenssl import inject_into_urllib3, extract_from_urllib3
+    pyopenssl_override = True
+except:
```

Comment:
what is the exact exception to be captured here?

Reply:
Based on this PR, looks like an `SSLError`, but I did it long time ago. Don't have time to investigate it again.